jenkins/job-configs/kubernetes-jenkins-pull/global.yaml

```diff
@@ -1,13 +1,28 @@
+- builder:
+    name: ensure-upload-to-gcs-script
+    builders:
+      - shell: |
+          _script_tmp_path='./_tmp/upload-to-gcs.sh'
+          _script_repo_path='hack/jenkins/upload-to-gcs.sh'
+          mkdir -p ./_tmp
+          if [[ "${{ghprbTargetBranch:-}}" == 'master' && -x "./kubernetes/${{_script_repo_path}}" ]]; then
```

Comment:
\neo_mouth_shut{If upload-to-gcs.sh is deleted}{How will anyone see the error?}

I'd originally written this to work for both post and PR Jenkins, but I can remove the check.

I might make everything relative to ${WORKSPACE}, too, since we do `cd` in the PR e2e job. I'm not sure if that would affect this at all, but probably best to be explicit?